src/renderer/shared/api/price-provider/common/constants.ts

```diff
@@ -0,0 +1,10 @@
+export const COINGECKO_URL = 'https://api.coingecko.com/api/v3';
+
+export const CURRENCY_URL =
+  'https://raw.githubusercontent.com/novasamatech/nova-wallet-android/develop/feature-currency-impl/src/main/res/raw/currencies.json';
```

Comment:
I suggest bundling the currency file with the application and not fetching it from the network. Otherwise, we might forget that Spektr depends on this source file in Nova Android and perform a breaking change that will break Spektr